### ISSUE TITLE
Wasm Runtime not included in vm_key

### ISSUE THREAD

**Issue by mathetake** [proxy-wasm/proxy-wasm-cpp-host#176]
this could be a problem once we support multiple runtimes because I think users would expect VMs to be created with a specified runtimes they change `vm_config.runtime`

**Comment by PiotrSikora** [comment]
Agreed, it should be included (although it's a no-op right now, since we only support a single runtime).
